bite-size-notes/week-6-c-state-estimation-intro.md

```diff
@@ -0,0 +1,193 @@
+# Deterministic State Estimation
+
+## Scope + Objectives
+
+These notes aim to provide readers with a thorough understanding and review of state estimation, the Luenberger Observer, and duality. The overall goals of these notes are to provide all background information necessary in the deterministic case to better understand the setup for stochastic state estimators such as the Kalman Filter.
+
+## Introduction
+
+So far in the course, we have considered states of a given system as given quantity in each iteration. However, in real systems, usually only the ouput $y(t)$ and input $u(t)$ are known quantities such that the actual state $x(t)$ needs to be computed. State estimators, also called observers, perform this computation to get an estimate of the state. 
+
+Why don't we know the actual state of the system? How can a determinisitic system still have errors? Usually, the first limiting factor when measuring the state is stochastic noise introduced into the measurement device which the Kalman filter attempts to statistically account for. However, in a determinisic system, there are still plenty of errors that arise to necessitate the need for a state estimator. Model inaccuracies, sensor limitations, parameter variations, and other non-linearities are all sources of error for deterministic state estimators.
+
+## Preliminaries
+
+First, we'll begin with a few definitions of relevant concepts used throughout the rest of the notes.
+
+### Relevant Definitions:
+
+- **Definition 1:**
+
+    A system is considered observable if, for any given time $T > 0$, it is possible to determine the system's state at that time $x(T) \in \mathbb{R}^n$ through measuring control input $u(t)$ and output $y(t)$ where $t \in [0,T]$
+
+    A pair of dynamics $(A,C)$ is observable if the observability matrix $O$ has $rank(O) = n = dim(A)$. The observability matrix is as follows:
+
+
+$$ \ O =    \begin{bmatrix}
+                C\\ CA \\\vdots \\ CA^{n-1}
+            \end{bmatrix}\ 
+$$
+
+- **Definition 2:**
+
+    The pair $(A, C)$ is detectable if all the unobservable modes are stable.
+
+- **Definition 3:**
+
+    A linear system is reachable if for any $x_o, x_f \in \mathbb{R}^n$, there exists a $T>0$ and $u(t): [0, T] \rightarrow \mathbb{R}$ such that the corresponding solution satisfies $x(0) = x_o$ and $x(T) = x_f \in \mathbb{R}^n$
+
+- **Definition 4:**
+
+    A system (A,B) is reachable if and only if $rank(R) = n$ where $x \in \mathbb{R}^n$ and
+
+    $$
+    R = \begin{pmatrix}
+    B & AB & \dots & A^{n-1}B
+    \end{pmatrix}
+    $$
+
+### Problem Definition:
+
+Given a linear, time invariant system ignoring noise, we have state and measurement output
+
```

Comment:
Could you make a line changing or a space between Bu(t) and y(t)?

Reply:
Fixed!